src/nbl/builtin/CMakeLists.txt

```diff
@@ -174,6 +174,8 @@ set(nbl_resources_to_embed
   "nbl/builtin/glsl/utils/normal_encode.glsl"
   "nbl/builtin/glsl/utils/transform.glsl"
   "nbl/builtin/glsl/utils/morton.glsl"
+  "nbl/builtin/glsl/utils/surface_transform.glsl"
+  "nbl/builtin/glsl/utils/surface_transform_e.h"
```

Comment:
Add `surface_transform_transformedDerivatives.glsl` here, you'll get glsl compile errors when it doesn't find it